MANIFEST.in

```diff
@@ -1,46 +1,22 @@
-recursive-include bootloader/src *
-recursive-include bootloader/tools *
-recursive-include bootloader/windows *
-recursive-include bootloader/zlib *
-include bootloader/README.txt
-include bootloader/waf
-include bootloader/wscript
-
-
-recursive-include doc/images *
-recursive-include doc/stylesheets *
-recursive-include doc *.1 *.html *.pdf *.rst *.txt
-
-recursive-include PyInstaller *
-
-recursive-include tests/functional *
-recursive-include tests/unit *
-recursive-include tests/old_suite *
-recursive-include tests *.txt
-
-recursive-include utils *.py
```

Comment:
FYI: I couldn't find this directory. It looks like it was deleted from the repository but not from here.